doc/classes/Basis.xml

```diff
@@ -444,11 +444,11 @@
 				Transforms (multiplies) the [param right] vector by this basis, returning a [Vector3].
 				[codeblocks]
 				[gdscript]
-				var my_basis = Basis(Vector3(1, 1, 1), Vector3(1, 1, 1), Vector3(0, 2, 5))
+				var my_basis = Basis(Vector3(1, -1, -1), Vector3(3, -1, 1), Vector3(0, 2, 5))
 				print(my_basis * Vector3(1, 2, 3)) # Prints (7, 3, 16)
 				[/gdscript]
 				[csharp]
-				var myBasis = new Basis(new Vector3(1, 1, 1), new Vector3(1, 1, 1), new Vector3(0, 2, 5));
+				var myBasis = new Basis(new Vector3(1, -1, -1), new Vector3(3, -1, 1), new Vector3(0, 2, 5));
 				GD.Print(my_basis * new Vector3(1, 2, 3)); // Prints (7, 3, 16)
```

Comment:
While you're changing this example, might as well fix this.

```suggestion
				GD.Print(myBasis * new Vector3(1, 2, 3)); // Prints (7, 3, 16)
```

Reply:
Good catch - i should not have skipped testing that

1. Changed Basis.xml as suggested
3. Ran `make rst`
4. Started ` sphinx-build.exe -b html ./ _build/html`
5. During build, opened the html file in the browser and confirmed it has the incorrect varname
6. After build, checked the browser again, it now had the correct varname
7. Installed godot with c# support and tested the script as copied from the browser, it worked as expected

Updated with --amend